src/normals.c

```diff
@@ -141,14 +144,15 @@ stl_fix_normal_directions(stl_file *stl) {
           (stl, stl->neighbors_start[facet_num].neighbor[j]);
         }
       }
+      const size_t neighbor_index = stl->neighbors_start[facet_num].neighbor[j];
```

Comment:
AFAIk admesh does not mix declarations and code. There used to be a Werror CI job but not since moving to CMake. No idea how to check that again.